tests/reg_tests/test_DVGeometryMulti.py

```diff
@@ -255,8 +255,8 @@ def twist(val, geo, nRefAxPts=nRefAxPts):
 
         # Check that the analytic derivatives are consistent with FD and CS
         for x in dvDict_real:
-            np.testing.assert_allclose(funcSens[x].T, funcSensFD[x], rtol=1e-4, atol=1e-10)
-            np.testing.assert_allclose(funcSens[x].T, funcSensCS[x], rtol=1e-4, atol=1e-10)
+            np.testing.assert_allclose(funcSens[x].T, funcSensFD[x], rtol=1e-4, atol=1e-7)
+            np.testing.assert_allclose(funcSens[x].T, funcSensCS[x], rtol=1e-4, atol=1e-7)
```

Comment:
I have to double check this, but from what I remember, this derivative test will be essentially useless with such a loose `atol`

Reply:
Yeah, the FD tests could be close... If that is the case, we either need platform dependent refs, or just not test on ARM for now. Let me know, but we can also discuss on https://github.com/mdolab/docker/pull/170 or offline.

Reply:
I checked this and there are entries in `funcSens` that are order 1e-8 and 1e-9. These are also the entries that are nonzero because of the intersection treatment, so loosening `atol` to 1e-7 means that we would not be testing the intersection derivatives. It's possible that the derivatives are wrong on the ARM image.

Reply:
I don’t have an M1/M2 unfortunately… @eytanadler @hajdik @awccopp do.

Reply:
@eytanadler is editing my comment to remove himself from this list…

Reply:
@hajdik too…

Reply:
I tried installing the necessary packages to test this locally but ran into problems compiling installing pySurf on ARM. I don't have Docker set up to test these easily, sorry.